xtask/src/commands/changeset/matching_pull_request.rs

```diff
@@ -0,0 +1,99 @@
+#![allow(clippy::all, warnings)]
+pub struct MatchingPullRequest;
+pub mod matching_pull_request {
+    #![allow(dead_code)]
+    use std::result::Result;
+    pub const OPERATION_NAME: &str = "MatchingPullRequest";
+    pub const QUERY : & str = "# This operation is used to generate Rust code which lives in a file directly\n# next to this with the same name but a `.rs` extension.  For instructions on\n# how to generate the code, see the top of `./mod.rs`.\nfragment PrInfo on PullRequest {\n  url\n  number\n  author {\n    __typename\n    login\n  }\n  title\n  closingIssuesReferences(last: 4) {\n    nodes {\n      url\n      number\n      repository {\n        nameWithOwner\n      }\n    }\n  }\n  body\n}\nfragment PrSearchResult on SearchResultItemConnection {\n  issueCount\n  nodes {\n    __typename\n    ...PrInfo\n  }\n }\n\nquery MatchingPullRequest($search: String!) {\n  search(\n    type: ISSUE\n    query: $search\n    first: 1\n  ) {\n    ...PrSearchResult\n  }\n}\n" ;
```

Comment:
include_str!("src/commands/changeset/matching_pull_request.graphql") maybe? 

EDIT: was this autogenerated? if so can we add a comment via the generator that says it was?

Reply:
The generator CLI — which is documented at the top of `mod.rs` doesn't allow adding a comment or making the change to `include_str`.

Since you're the second person who has asked about this and suggested `include_str!`, and I don't think the generated code is ever going to change, I'm going to just add a comment at the top of this file saying it's generated.  Worst case, we lose that comment in the future.  Best case, someone understands how to re-generate it and doesn't modify it manually.

Reply:
Done with 4e793352